<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>2</y>
     <width>120</width>
     <height>76</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>76</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>76</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_filename">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>0</y>
     <width>170</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#label_filename {
    color:rgb(255,255,255);
    background-color: transparent;
    font-family: &quot;新宋体&quot;;
    font-size: 17px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_duration">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>55</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#label_duration {
    color:rgb(255,255,255);
    background-color: transparent;
    font-family: &quot;新宋体&quot;;
    font-size: 15px;
	font-weight:bold;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_remove">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>32</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
